tests/countries/test_anguilla.py

```
@@ -0,0 +1,295 @@
+            "2004-06-01",
+            "2009-06-02",
+            "2010-05-31",
+            "2015-06-01",
+            "2020-06-02",
+            "2021-05-31",
+            "2039-05-31",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_august_monday_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)